js/src/3rdparty/sms-verification/index.js

```diff
@@ -15,17 +15,6 @@
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
 import { stringify } from 'querystring';
-import React from 'react';
-
-export const termsOfService = (
```

Comment:
@jacogr 

I put this here on purpose. The terms of service describe the legal implications of using the 3rd party service in `3rdparty/sms-verification`. I can understand that having React in here feels weird, but since `termsOfService` is stateless & static, I think it's worth it. Having it in `modals/SMSVerification/GatherData` is very unintuitive imho.